### PR TITLE
JDBC-59 Publish BI Tool dialects for JDBC connectivity

### PR DESCRIPTION
**Pull request by jpmcmu**
Signed-off-by: James McMullan James.McMullan@lexisnexis.com

## Type of change:
- [X] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).

## Checklist:
- [X] I have created a corresponding JIRA ticket for this submission
- [X] My code follows the code style of this project.
  - [ ] I have applied the Eclipse code-format template provided.
- [X] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [X] I have read the HPCC Systems CONTRIBUTORS document (https://github.com/hpcc-systems/HPCC-Platform/wiki/Guide-for-contributors).
- [X] The change has been fully tested:
  - [ ] I have performed unit tests to cover my changes.
  - [ ] I have performed system test and covered possible regressions and side effects.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [X] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.

### CODE REVIEW

**Comment by jpmcmu** [comment]
@rpastrana please review

**Review by rpastrana — COMMENTED** [review]
@jpmcmu looks good, let's add a simple readme

**Comment by jpmcmu** [comment]
@rpastrana updated please review

File: bitools/tableau/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Tableau Resources
+=======================
+
+#### Description
+These files provide the needed configuration to connect to HPCC from Tableau. The hpcc-jdbc.tdc file specifies the basic functionality and driver type for the jdbc connection to HPCC. The example.properties file is a template for a connnection properties file; this template file should be used to create a new properties for each HPCC cluster that you Tableau will connect to. These connection files can be placed anywhere on your system.
```
Comment (rpastrana) [inline bitools/tableau/README.md:5]: you => your

File: bitools/tableau/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+hpcc-jdbc.tdc should be placed in the `$DOCUMENTS\My Tableau Repository\Datasources` directory. 
+
+#### Connecting to HPCC via JDBC
+Install the built jdbc jar from this project into the `$Tableau/Drivers` directory. Start Tableau select: Other Databases (JDBC). In the pop-up window type `jdbc:hpcc` into the URL field, under dialect select MySQL, provide your HPCC credentials, and select the properties file you created above for the HPCC cluster you wish to connect to.
```
Comment (rpastrana) [inline bitools/tableau/README.md:10]: Nitpick, does it make sense to reference the jar by it's official name? or the HPCC jdbcdriver.jar ?
Also, can we split this into a bulleted list?

**Review by rpastrana — COMMENTED** [review]
@jpmcmu looks good, just a couple of comments.

**Comment by jpmcmu** [comment]
@rpastrana updated please review

**Review by rpastrana — APPROVED** [review]

**Pull request merged**